Runtime/Visual/CameraColorOverlay.cs

```diff
@@ -252,5 +269,23 @@ protected virtual void PostRender(Camera cam)
                 GL.PopMatrix();
             }
         }
+
+        /// <summary>
+        /// Copies the <see cref="OverlayMaterial"/> data to the <see cref="workingMaterial"/> data.
+        /// </summary>
+        protected virtual void CopyMaterialOverlayToWorking()
+        {
+            workingMaterial = new Material(OverlayMaterial);
```

Comment:
This needs to destroy the previous `workingMaterial` before creating a new instance (if it's not `null` that is).

Reply:
Why do we need to destroy the previous one? Won't this simply overwrite the new one and the old one will be garbage collected?

Reply:
No, because `Material` is a Unity "resource" and as such needs to be destroyed on their native engine side.

Reply:
(It shows that by inheriting from `UnityEngine.Object` - all of those need to be `Destroy`ed.)